jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/HttpConnectionTest.java

```diff
@@ -1724,4 +1729,97 @@ public boolean handle(Request request, Response response, Callback callback) thr
         else
             assertThat(response.get(HttpHeader.CONNECTION), is(expectedConnectionHeader));
     }
+
+    @Test
+    public void testRetainedChunks() throws Exception
```

Comment:
This test should be moved to `jetty-test-client-transport` to test it with all transports (apart FCGI).